projects/sandbox/data_generation/data_generation/generate_testing_set.py

```diff
@@ -33,6 +37,7 @@ def main(
     min_duration: float = 0,
     waveform_arguments: Optional[Dict] = None,
     parameter_conversion: Optional[Callable] = None,
+    gaussian: bool = False,
```

Comment:
update docstring.

Reply:
Got it will update doc string. 

I use the requested period of real data to compute the psd, so no need to specify a path.

Reply:
OK, but we can very likely be asked to compare posteriors across different observing runs, and possibly next gen instruments. Besides the changes to achieve that is pretty minimal compared to what you have here.

Reply:
Okay are you thinking of adding an argument like 

```python
psd: Optional[Path] = None
```
If this is set and `gaussian = True`, we use for the psd, otherwise we use the requested data

Reply:
Yes exactly, a path and a `is_asd` which is a bool, since many of the psd files are actually asds.